src/test/java/nextstep/courses/domain/SessionTest.java

```diff
@@ -0,0 +1,57 @@
+package nextstep.courses.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class SessionTest {
+
+    public static final SessionPeriod PERIOD = new SessionPeriod(LocalDate.now(), LocalDate.now().plusDays(1L));
+    public static final SessionCoverImage COVER_IMAGE = new SessionCoverImage(500 * 1024, "jpg", 300, 200);
+    public static final SessionAmount AMOUNT = new SessionAmount(100_000L);
+
+    public static final SessionAddInfo ADD_INFO = new SessionAddInfo(0L, 100_000L);
+    public static final SessionAddInfo DISMATCH_ADD_INFO = new SessionAddInfo(0L, 10_000L);
+
+    private Session paidSession;
+
+    @BeforeEach
+    void setUp() {
+        paidSession = Session.createPaidSession(0L, PERIOD, COVER_IMAGE, AMOUNT, 1, SessionStatus.RECRUITING);
```

Comment:
```suggestion
        paidSession = Session.paidSession(0L, PERIOD, COVER_IMAGE, AMOUNT, 1, SessionStatus.RECRUITING);
```
팩토리 메소드이므로 무엇을 생성하는 행위가 기본적으로 깔려있으니 create 정도는 생략해도 될 것 같아요
저의 개인적인 의견이므로 참고만 해주세요~